### PR TITLE
feat: add Container primitive

### PR DESCRIPTION
**Pull request by martapa**
## Purpose
- Add Container primitive 
- Add story for Container
## Screenshots
image

## Checklist

- [ ] Title follows the [naming convention](../README.md#pull-requests).
- [ ] Add a description.
- [ ] Add screenshots (optional).
- [ ] Add testing instructions.
- [ ] Add at least one reviewer.
- [ ] Apply related labels (optional).
- [ ] Leave comments on changes you would like to discuss.
- [ ] Remove changes that are not related to this pull request.
- [ ] Remove debugging code.
- [ ] Test your changes locally and on staging (if applicable).
- [ ] Ensure test coverage value in config matches actual coverage (if applicable).
- [ ] Ensure the build is passing.

### CODE REVIEW

**Review by karlpawlowicz — APPROVED** [review]

File: src/blocks/Container.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return (
+    <div className={className} ref={ref}>
+      {children}
+    </div>
+  )
+}
+
+Container.displayName = 'Container'
+
+export default React.forwardRef<HTMLParagraphElement, ComposableProps>(
```
Comment (mattxyzeth) [inline src/blocks/Container.tsx]: Should be `HTMLDivElwment` for the type.

File: src/blocks/Container.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  hooks: {},
+  props: {},
+  state: {},
+  children: [],
+  className: 'string',
+  type: 'Container',
+}
+
+const Container: React.ForwardRefRenderFunction<
+  HTMLParagraphElement,
```
Comment (mattxyzeth) [inline src/blocks/Container.tsx]: Should be `HTMLDivElement` for the type.

**Review by mattxyzeth — APPROVED** [review]
Update those types and then ship it. :shipit:

File: src/blocks/Container.tsx (hunk 1 above)
Comment (martapa) [reply src/blocks/Container.tsx]: 💩 oops

**Pull request merged**